releases/release-5.0.6.md

```diff
@@ -0,0 +1,169 @@
+    (dup) - Fix the issue that the `GREATEST` function returns inconsistent results due to different values of `tidb_enable_vectorized_expression` (`on` or `off`) [#29434](https://github.com/pingcap/tidb/issues/29434)
+    (dup) - Fix the issue that planner might cache invalid plans for `join` in some cases [#28087](https://github.com/pingcap/tidb/issues/28087)
+    - Fix the issue that the `index out of range [1] with length 1` error is raised when a SQL statement tries to evaluate an aggregation result on the result of join in some cases [#1978](https://github.com/pingcap/tics/issues/1978)
+
++ TiKV
+
+    - Fix the issue that a downed TiKV node causes the resolved timestamp to lag behind [#11351](https://github.com/tikv/tikv/issues/11351)
+    (dup) - Fix the issue that batch messages are too large in Raft client implementation [#9714](https://github.com/tikv/tikv/issues/9714)
+    (dup) - Fix a panic issue that occurs when Region merge, ConfChange, and Snapshot happen at the same time in extreme conditions [#11475](https://github.com/tikv/tikv/issues/11475)
+    - Fix the issue that TiKV cannot detect the memory lock when `TableScan` is in descending order [#11440](https://github.com/tikv/tikv/issues/11440)
+    (dup) - Fix the issue of negative sign when the decimal divide result is zero [#29586](https://github.com/pingcap/tidb/issues/29586)
+    - Fix the issue that the accumulation of GC tasks might cause TiKV to be OOM (out of memory) [#11410](https://github.com/tikv/tikv/issues/11410)
+    (dup) - Fix the issue that the average latency of the by-instance gRPC requests is inaccurate in TiKV metrics [#11299](https://github.com/tikv/tikv/issues/11299)
+    (dup) - Fix a memory leak caused by monitoring data of statistics threads [#11195](https://github.com/tikv/tikv/issues/11195)
+    (dup) - Fix the issue of TiCDC panic that occurs when the downstream database is missing [#11123](https://github.com/tikv/tikv/issues/11123)
+    (dup) - Fix the issue that CDC adds scan retries frequently due to the Congest error [#11082](https://github.com/tikv/tikv/issues/11082)
+    (dup) - Fix the issue that the Raft connection is broken when the channel is full [#11047](https://github.com/tikv/tikv/issues/11047)
+    - Fix the issue of TiKV panic that occurs when the files do not exist when Lightning importing data [#10438](https://github.com/tikv/tikv/issues/10438)
+    (dup) - Fix the issue that TiDB cannot correctly identify whether the `Int64` types in `Max`/`Min` functions are a signed integer or not, which causes the wrong calculation result of `Max`/`Min` [#10158](https://github.com/tikv/tikv/issues/10158)
+    - Fix the issue that the nodes of a TiKV replica are down after the nodes get snapshots because TiKV cannot modify the meta information accurately [#10225](https://github.com/tikv/tikv/issues/10225)
```

Comment:
```suggestion
    - Fix the issue that the node of a TiKV replica is down after the node gets snapshots because TiKV cannot modify the metadata accurately [#10225](https://github.com/tikv/tikv/issues/10225)
```